src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverProjectExtension.kt

```diff
@@ -141,6 +141,23 @@ public interface KoverProjectExtension {
     }
 }
 
+/**
+ * Disables instrumentation of test tasks.
+ *
+ * This means that even if the excluded test is executed, the function calls that occurred in it will not be counted in the reports.
+ *
+ * As a side effect, reports cease to depend on the specified test tasks.
+ *
+ * Example:
+ * ```
+ *     kover {
+ *         excludeTests {
+ *             tasks("test1", "test2")
+ *             mppTargetName("jvm")
+ *         }
+ *     }
+ * ```
+ */
 public interface KoverTestsExclusions {
```

Comment:
Here the documentation to functions `tasks` states "Disables instrumentation of specified tests.", but it's not in fact true -- it disables the corresponding test tasks by their name. It should be reflected in the doc